### ISSUE TITLE
Packaging: wheel not available, build from source fails

### ISSUE THREAD

**Issue by mmore500** [emilydolson/python-phylogeny-tracker#81]
**Describe the bug**
Wheel is not available on HPCC. Fallback to build from source fails

**To Reproduce**
```bash
module purge || :
module load GCCcore/12.2.0 Python/3.10.8 || :
python3 -m venv env
source env/bin/activate
python3 -m pip install "phylotrackpy==0.2.0"
```
```
Collecting phylotrackpy==0.2.0
  Using cached phylotrackpy-0.2.0.tar.gz (18 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Building wheels for collected packages: phylotrackpy
  Building wheel for phylotrackpy (pyproject.toml) ... error
  error: subprocess-exited-with-error
  
  × Building wheel for phylotrackpy (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [16 lines of output]
      running bdist_wheel
      running build
      running build_py
      creating build
      creating build/lib.linux-x86_64-cpython-310
      creating build/lib.linux-x86_64-cpython-310/phylotrackpy
      copying phylotrackpy/__init__.py -> build/lib.linux-x86_64-cpython-310/phylotrackpy
      running build_ext
      building 'phylotrackpy.systematics' extension
      creating build/temp.linux-x86_64-cpython-310
      gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -O2 -ftree-vectorize -march=x86-64 -mtune=generic -fno-math-errno -fPIC -O2 -ftree-vectorize -march=x86-64 -mtune=generic -fno-math-errno -fPIC -fPIC -DVERSION_INFO=0.2.0 -DEMP_OPTIONAL_THROW_ON=1 -I/tmp/pip-build-env-s4a73k8e/overlay/lib/python3.10/site-packages/pybind11/include -I/mnt/ufs18/home-089/mmore500/env310-/include -I/opt/software/Python/3.10.8-GCCcore-12.2.0/include/python3.10 -c systematics_bindings.cpp -o build/temp.linux-x86_64-cpython-310/systematics_bindings.o -fvisibility=hidden -g0 -std=c++20
      systematics_bindings.cpp:12:10: fatal error: Empirical/include/emp/Evolve/Systematics.hpp: No such file or directory
         12 | #include "Empirical/include/emp/Evolve/Systematics.hpp"
            |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      compilation terminated.
      error: command '/opt/software/GCCcore/12.2.0/bin/gcc' failed with exit code 1
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for phylotrackpy
Failed to build phylotrackpy
ERROR: Could not build wheels for phylotrackpy, which is required to install pyproject.toml-based projects

[notice] A new release of pip available: 22.2.2 -> 24.0
[notice] To update, run: pip install --upgrade pip
```

**Expected behavior**
The necessary  wheel to work with the HPCC should be deployed to PyPi.
Additionally, the Empirical source should be packaged into the source distribution on PyPi so that fallback source builds do not fail

**Computational environment (please complete the following information):**
 - OS: [e.g. iOS] dev-amd20 centOS
 - Python version [e.g. 3.8] 3.10

**Additional context**
This workaround to install directly from GitHub works
```bash
python3 -m pip install git+https://github.com/emilydolson/phylotrackpy@v0.2.0
```

**Comment by mmore500** [comment]
Note that `python3 -m pip install phylotrackpy==0.2.0 --binary-only=phylotrackpy` also fails to install wheel. install `--upgrade`'ing pip and wheel do not help.

**Comment by mmore500** [comment]
We should probably add tests to CI a la 

<https://github.com/mmore500/hstrat/blob/ba7b19d0f0561303b6bf789d5024a323aa5993fc/.github/workflows/ci.yaml#L201>

**Comment by emilydolson** [comment]
Looks like things are working on the HPCC now (at least on my account). sdist build is fixed and regression test added in #99

**Comment by mmore500** [comment]
:tada:
